modules/polygon/src/index.js

```diff
@@ -1,5 +1,18 @@
 export {default as Polygon} from './polygon';
 
+export {
+  getPolygonSignedArea,
+  getPolygonSignedAreaFlat,
+  getPolygonWindingDirection,
+  getPolygonWindingDirectionFlat,
+  forEachSegmentInPolygon,
+  forEachSegmentInPolygonFlat,
+  ensurePolygonWindingDirection,
+  ensurePolygonWindingDirectionFlat,
+  WINDING_CLOCKWISE,
+  WINDING_COUNTER_CLOCKWISE
+} from './polygon-utilities';
```

Comment:
Nit: Looking at other code, `polygon-utils.js` seems to be more common https://github.com/uber-web/math.gl/blob/master/modules/core/src/lib/math-utils.js